decidim-core/app/packs/src/decidim/editor/extensions/decidim_kit/index.js

```diff
@@ -3,12 +3,11 @@ import { Extension } from "@tiptap/core";
 import StarterKit from "@tiptap/starter-kit";
 import CodeBlock from "@tiptap/extension-code-block";
 import Underline from "@tiptap/extension-underline";
+import Heading from "@tiptap/extension-heading";
```

Comment:
I think this is an unused import, as it isn't being referenced. Can you remove it? 
```suggestion
```

Reply:
@andreslucena actually I think the default needed if I'm not mistaken to replace the deleted `decidim-core/app/packs/src/decidim/editor/extensions/heading/index.js`.

Reply:
As far as I see, this import isn't referenced anymore in the JS file. Can you remove the line to see if everything passes?

Reply:
@andreslucena checkout the pipeline....

Reply:
Reverting due to failures:

![image](https://github.com/user-attachments/assets/52d760f4-8708-4723-96d6-bd8778824035)